falcon-regression/merlin-core/src/main/java/org/apache/falcon/regression/core/util/InstanceUtil.java

```diff
@@ -516,7 +516,7 @@ public static String getOutputFolderForInstanceForReplication(OozieClient oozieC
     private static String getReplicatedFolderFromInstanceRunConf(String runConf) {
         String inputPathExample = getReplicationFolderFromInstanceRunConf(runConf).get(0);
         String postFix = inputPathExample.substring(inputPathExample.length() - 7, inputPathExample.length());
-        return getReplicatedFolderBaseFromInstanceRunConf(runConf) + postFix;
+        return getReplicatedFolderBaseFromInstanceRunConf(runConf) + "/" + postFix;
     }
```

Comment:
Cannot use this function. It replaces all the occurrences of '//' by '/'. My input contains hdfs://randomPath which is converted to hdfs:/randomPath which is not the expected value.